### PR TITLE
Create and Delete user admin draft

### PR DESCRIPTION
**Pull request by BogdanQA**
https://trello.com/c/1SiQF4UI
https://trello.com/c/T86mEEda

### CODE REVIEW

**Review by guram21 — APPROVED** [review]

**Review by Burdishka — APPROVED** [review]

File: src/test/java/model/admin/table/UsersTablePage.java [hunk 1]
```diff
@@ -35,4 +59,58 @@ protected UserEditPage createAdminEditPage() {
...
+
+    public boolean successfulUserCreationMessage() {
+        WebElement confirmation = getDriver().findElement(CONFIRM);
+        return confirmation.isDisplayed();
+    }
+
+    public UserViewPage viewUser() {
+        TestUtils.jsClick(getDriver(), viewUser);
+        return new UserViewPage(getDriver());
+    }
```
Comment (oooleks) [inline src/test/java/model/admin/table/UsersTablePage.java:75]: You have to use AdminBaseTablePage.viewRow() method instead

File: src/test/java/model/admin/table/UsersTablePage.java [hunk 2]
```diff
@@ -35,4 +59,58 @@ protected UserEditPage createAdminEditPage() {
...
+
+    public UserEditPage editUser() {
+        TestUtils.jsClick(getDriver(), editUser);
+        return new UserEditPage(getDriver());
+    }
```
Comment (oooleks) [inline src/test/java/model/admin/table/UsersTablePage.java:80]: You have to use AdminBaseTablePage.editRow() method instead

File: src/test/java/model/admin/table/UsersTablePage.java [hunk 3]
```diff
@@ -35,4 +59,58 @@ protected UserEditPage createAdminEditPage() {
...
+
+    public UsersTablePage deleteUser() {
+        TestUtils.jsClick(getDriver(), deleteUser);
+        return this;
+    }
```
Comment (oooleks) [inline src/test/java/model/admin/table/UsersTablePage.java:85]: You have to use AdminBaseTablePage.deleteRow() method instead

File: src/test/java/model/admin/table/UsersTablePage.java [hunk 4]
```diff
@@ -1,18 +1,42 @@
...
+    private WebElement viewUser;
+
+    @FindBy(xpath = "//a[contains(text(),'edit')]")
+    private WebElement editUser;
+
+    @FindBy(xpath = "//a[contains(text(),'delete')]")
+    private WebElement deleteUser;
+
+    @FindBy(className = "card-body")
+    private WebElement body;
```
Comment (oooleks) [inline src/test/java/model/admin/table/UsersTablePage.java:35]: These elements already present in extended Base page, hence should be deleted

File: src/test/java/model/admin/table/UsersTablePage.java [hunk 5]
```diff
@@ -1,18 +1,42 @@
...
+
+    @FindBy(xpath = "//table[@id='pa-all-entities-table']/tbody/tr")
+    private List<WebElement> trs;
```
Comment (oooleks) [inline src/test/java/model/admin/table/UsersTablePage.java:38]: This element is already present in extended AdminBaseTablePage page, hence should be deleted

**Pull request merged**